subsys/bluetooth/mesh/transport_legacy.c

```diff
@@ -870,11 +870,9 @@ static int trans_ack(struct bt_mesh_net_rx *rx, uint8_t hdr,
 	}
 
 	if (tx->nack_count) {
-		/* According to the Bluetooth Mesh Profile specification,
-		 * section 3.5.3.3, we should reset the retransmit timer and
-		 * retransmit immediately when receiving a valid ack message.
-		 * Don't reset the retransmit timer if we didn't finish sending
-		 * segments.
+		/* According to MshPRT-v1.0.1: 3.5.3.3, we should reset the retransmit timer and
```

Comment:
So here it is 1.0.1, not 1.1?

Reply:
Since this is the legacy transport it made sence to reference 1.0.1. The legacy transport is for the 1.0.1 spec only right?

Reply:
Ah, true, but then it should be MshPRFv1.0.1, not MshPRTv1.0.1